Directory.Build.props

```diff
@@ -7,13 +7,10 @@
   <PropertyGroup>
     <LangVersion>Latest</LangVersion>
     <Features>strict</Features>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
```

Comment:
This was intentional.  We do not want overflow/underflow to interfere with the library but I did want actual overflow issues to surface in test runs or when debugging the library.  Was this changed on purpose?

Reply:
I'll revert this.

Reply:
The current configurations override this property to always false in non-test libraries.

Reply:
Enabling overflow checks broke a bunch of tests 😅. That's a real functionality change which should be saved for later.

Reply:
haha that's pretty interesting...  I should look into that at some point.